archived/event-adapter/adapter.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: event-adapter
   namespace: default
 ---
-apiVersion: extensions/v1beta1
```

Comment:
I don't think things in `archived` should be updated

Reply:
thanks, removed

Reply:
ready for another review now